web/wiki/Porting.md

```diff
@@ -30,15 +30,21 @@ Identifier           | Platform(s)                | Usage
 ---------------------|----------------------------|----------------------------
 `_WIN32`             | mingw, msvc                | C code (`#ifdef ...`)
 `_WIN64`             | 64-bit mingw, 64-bit msvc  | C code (`#ifdef ...`)
+`__MINGW32__`        | mingw                      | C code (`#ifdef ...`)
+`__MINGW64__`        | 64-bit mingw               | C code (`#ifdef ...`)
 `__CYGWIN__`         | msys2, cygwin              | C code (`#ifdef ...`)
 `__MSYS__`           | msys2                      | C code (`#ifdef ...`)
-`x86_64-pc-msys2`    | 64-bit msys2               | Build scripts (`if [ $host = '...' ]`)
-`i686-pc-msys2`      | 32-bit msys2               | Build scripts (`if [ $host = '...' ]`)
+`x86_64-pc-msys`     | 64-bit msys2 (old style)   | Build scripts (`if [ $host = '...' ]`)
+`i686-pc-msys`       | 32-bit msys2 (old style)   | Build scripts (`if [ $host = '...' ]`)
+`x86_64-pc-cygwin`   | 64-bit msys2 (new style)   | Build scripts (`if [ $host = '...' ]`)
+`i686-pc-cygwin`     | 32-bit msys2 (new style)   | Build scripts (`if [ $host = '...' ]`)
 `x86_64-w64-mingw32` | 64-bit mingw               | Build scripts (`if [ $host = '...' ]`)
```

Comment:
You can add aarch64-\* targets :-D.

Reply:
Feel free to! You probably know those much better.